java-frontend/src/main/java/org/sonar/java/se/xproc/MethodYield.java

```diff
@@ -175,6 +179,15 @@ public boolean equals(Object obj) {
     if(node == null || behavior == null) {
       return Collections.emptySet();
     }
+    String key = parameterIndices.stream().sorted().map(Object::toString).reduce("", String::concat);
```

Comment:
I'm not sure about that generating the key like this will be unique: from how I see it,
if we call the method with [1,2], we will end up with the same key (`"12"`) as asking for the 12th parameter with the list [12]. (I know this is pretty unlikely, but still...)